Gemfile.rails-5.0.0

```diff
@@ -0,0 +1,6 @@
+source "https://rubygems.org"
+
+gem 'rails', '5.0.0'
```

Comment:
Sure, that's fine. We can improve it after if you want.

It's just redundant right now because the default gemspec/Gemfile will resolve to `5.0.0` which covers `Gemfile.rails-5.0.0`.